client/resource_group/controller/limiter.go

```diff
@@ -371,7 +371,7 @@ func (lim *Limiter) reserveN(now time.Time, n float64, maxFutureReserve time.Dur
 		lim.tokens = tokens
 		lim.maybeNotify()
 	} else {
-		log.Warn("[resource group controller]", zap.Float64("NewTokens", lim.tokens), zap.Float64("current rate", float64(lim.limit)), zap.Float64("request tokens", n), zap.Int64("burst", lim.burst), zap.Int("remaining notify times", lim.remainingNotifyTimes))
+		log.Debug("[resource group controller]", zap.Float64("Tokens", lim.tokens), zap.Float64("current rate", float64(lim.limit)), zap.Float64("request tokens", n), zap.Int64("burst", lim.burst), zap.Int("remaining notify times", lim.remainingNotifyTimes))
```

Comment:
How about using `-` to link the words.

Reply:
@CabinfeverB Can u also update this line? :) https://github.com/tikv/pd/blob/master/pkg/mcs/resource_manager/server/grpc_service.go#L217